### DISCUSSION TITLE
【任务（已完成）】教程翻译计划：英文｜15,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#117] [任务 | Bounty]
WTF Solidity极简教程目前已有40讲，有国外的朋友问是否能翻译成英文，扩大受众。这个任务是为了实验将教材以去中心化的协作方式翻译成英文是否可行。

**任务**：翻译WTF Solidity极简教程1-10讲的教程和代码注释翻译成英文。

**赏金**：15,000 $PEOPLE 代币，根据工作量分配。

**规则**：此任务面向WTF社群成员和LanguageDAO成员开放。翻译后的教程归档至WTFSolidity github repo的Languages/en目录下，文件夹名后加_en后缀：例如`01_HelloWeb3`文件夹改为`01_HelloWeb3_en`，其他文件名不变。请保证翻译质量，会有贡献者审核。

有用的资料：
1. Markdown书写规则：https://www.markdownguide.org/cheat-sheet/
2. 审核并添加图片后的例子：https://github.com/AmazingAng/WTFSolidity/tree/main/02_ValueTypes
3. 审核第一期：https://github.com/AmazingAng/WTFSolidity/discussions/18
4. 已翻译好的教程：https://github.com/AmazingAng/WTFSolidity/tree/main/Languages/en

### 认领情况：
认领情况：
第1讲：三行代码HelloWeb3
@quantum-proof from LanguageDAO  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/116

第2讲：数值类型
@quantum-proof from LanguageDAO  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/116

第3讲：函数类型 (external/internal/public/private, pure/view/payable)
[lcy101u](https://github.com/lcy101u) **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/140

第4讲：函数输出（returns/return）
[reborn-sama](https://github.com/reborn-sama)  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/127

第5讲：变量作用域和数据存储 (storage/memory/calldata)
[buttonwild](https://github.com/buttonwild)  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/124

第6讲：数组（array）和结构体（struct）
[buttonwild](https://github.com/buttonwild)  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/131

第7讲：映射（mapping）
[seasidejuvenile666](https://github.com/seasidejuvenile666)  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/119

第8讲：变量初始值
[seasidejuvenile666](https://github.com/seasidejuvenile666)  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/119

第9讲：常量（constant/immutable）
[reborn-sama](https://github.com/reborn-sama)  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/127

第10讲：控制流和插入排序
@JustinAsdz **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/135

80%给翻译者，20%给审核者（@quantum-proof审核了7讲，0xAA审核了3讲）。
收据：https://etherscan.io/tx/0x437de6f6247340e5e23ee3d02b9beb3d6549ab0279b127807db61bff7e37722d
发放日期：2022.08.06

**Comment by buttonwild** [comment]
5，6

**Comment by lcy101u** [comment]
3

**Comment by reborn-sama** [comment]
4, 9

**Comment by seasidejuvenile666** [comment]
7、8

**Comment by JustinAsdz** [comment]


**Reply by buttonwild** [reply]
1，2已被认领，可看上方认领情况。

**Reply by JustinAsdz** [reply]
> 1，2已被认领，可看上方认领情况。

1 那是剩了个10吗 我认领10把

**Comment by JustinAsdz** [comment]
10

**Comment by joyc** [comment]
请问能否申请或参与翻译至其他语言版本。
